package.json

```diff
@@ -10,11 +10,22 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@fontsource/roboto": "^5.0.12",
     "@fortawesome/fontawesome-free": "^6.5.2",
+    "@mui/icons-material": "^5.15.15",
+    "@mui/material": "^5.15.15",
+    "@types/material-ui": "^0.21.17",
+    "axios": "^1.6.8",
     "bootstrap": "^5.3.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.22.3"
+    "react-hook-form": "^7.51.2",
+    "react-loader-spinner": "^6.1.6",
```

Comment:
We don't need a package for loaders, you can look for MUI's loaders in these two pages:

https://mui.com/material-ui/react-backdrop/
https://mui.com/material-ui/react-progress/